src/api/app/components/add_review_dropdown_component.html.haml

```diff
@@ -2,15 +2,16 @@
   .btn.btn-success.toggle-review-form{ type: 'button', data: { toggle: 'collapse', target: '#review-form-collapse',
                                        review: @my_open_reviews.first.id, reviewer_icon: reviewer_icon_and_text(review: @my_open_reviews.first) },
                                        aria: { expanded: 'false', controls: 'review-form-collapse' } }
-    Review
+    Review as
+    = @my_open_reviews.first.reviewer
```

Comment:
I don't like this UX wise. This gives the role more emphasis than it has, the important part is giving the review. That a users has 2 reviews open for different roles is a corner case anyway...

Reply:
For the sake of recognition I would have the button say the same thing in all circumstances that that is "Review"

Reply:
done